fairlearn/metrics/_fairness_metrics.py

```diff
@@ -165,9 +177,9 @@ def equalized_odds_ratio(
     y_pred,
     *,
     sensitive_features,
-    method="between_groups",
+    method: Literal["between_groups", "to_overall"] = "between_groups",
     sample_weight=None,
-    agg="worst_case",
+    agg: Literal["worst_case", "mean"] = "worst_case",
```

Comment:
I believe the the docstring can be copied in all the relevant places, just mentioning these two here as well to not be missed.